source/DistanceAndDirection/ProAppDistanceAndDirectionModule/ProAppDistanceAndDirectionModule/ViewModels/ProRangeViewModel.cs

```diff
@@ -189,7 +189,8 @@ private void DrawRadials()
                         MapPoint movedMP = null;
                         var mpList = new List<MapPoint>() { Point1 };
                         // get point 2
-                        var tempRadialLength = ConvertFromTo(LineDistanceType, DistanceTypes.Meters, radialLength);
+                        //var tempRadialLength = ConvertFromTo(LineDistanceType, DistanceTypes.Meters, radialLength);
+                        var tempRadialLength = radialLength;
```

Comment:
Is tempRadialLength needed/used anymore? If not, it can be removed and/or commented if just a temp change